components/form/form-element-mixin.js

```diff
@@ -195,6 +204,11 @@ export const FormElementMixin = superclass => class extends LocalizeCoreElement(
 		if (this.noValidate) {
 			return [];
 		}
+
+		// validation can be requested before first update, which can cause issues
+		// if validation logic depends on rendered DOM
+		await this._firstUpdatePromise;
```

Comment:
I originally thought I could use `await this.updateComplete` but unfortunately:
> If `shouldUpdate()` returns true, which it does by default, then the update proceeds normally. If it returns false, the rest of the update cycle will not be called but the `updateComplete` Promise is still resolved.

We're using `shouldUpdate` in our localization mixins to prevent rendering until the resources have loaded. But this weird behaviour here means that `updateComplete` can still resolve before the translations, which I was seeing here -- hence using my own Promise to track it.

I had to copy this same logic below into `form-mixin`... if this ends up being "a thing" we may want to factor it out into its own mixin. I'm surprised Lit doesn't provide something like this TBH.